app/uk/gov/hmrc/trackingconsentfrontend/assets/test/ui/renderBanner.spec.ts

```diff
@@ -59,13 +60,13 @@ describe('renderBanner', () => {
   }
 
   it('should render a banner', () => {
-    renderBanner(userPreference)
+    renderBanner(document, userPreference)
```

Comment:
It would be good to create a JSDOM for each test like https://github.com/hmrc/tracking-consent-frontend/blob/master/app/uk/gov/hmrc/trackingconsentfrontend/assets/test/ui/settingsFormHandler.spec.ts#L24 and remove the need for line 41 as a reset.

Reply:
I think we should move away from creating a new JSDOM for each test - this is causing our tests to run very slowly.

Reply:
In that case I think we should use a different mock/stub of document where we mock out the functions we need.  The shared scope between unit tests is really worth avoiding.

Reply:
OK, but this is not really relating to this PR as the tests haven't changed. Can we address this later on?